src/main/java/com/elytradev/concrete/inventory/gui/widget/Widget.java

```diff
@@ -33,9 +33,8 @@
 import net.minecraftforge.fml.relauncher.Side;
 import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class WWidget {
+public class Widget {
```

Comment:
It should also be noted that most (if not all) Swing components extend the `Component` class, which does not have the `J` prefix.